main/gdk-pixbuf/APKBUILD

```diff
@@ -1,22 +1,19 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=gdk-pixbuf
-pkgver=2.36.11
-pkgrel=2
+pkgver=2.38.1
+pkgrel=0
 pkgdesc="GTK+ image loading library"
 url="https://www.gtk.org/"
```

Comment:
https://wiki.gnome.org/Projects/GdkPixbuf

Reply:
Thanks, missed that somehow.